packages/astro/test/astro-attrs.test.js

```diff
@@ -55,6 +55,13 @@ describe('Attributes', async () => {
 		expect($('img').attr('happy:smile')).to.equal('sweet');
 	});
 
+	it('Passes namespaced expression attributes as expected', async () => {
+		const html = await fixture.readFile('/namespaced-with-expression/index.html');
+		const $ = cheerio.load(html);
+
+		expect($('use').attr('xlink:href')).to.deep.equal('#test');
```

Comment:
cheerio somehow only sees `href`, not `xlink:href`. I tried with jsdom and the tests pass.
Also when `xlink:href` is set in a div, tests pass